seatunnel-common/src/main/java/org/apache/seatunnel/common/utils/ReflectionUtils.java

```diff
@@ -0,0 +1,41 @@
+/*
+ * Licensed to the Apache Software Foundation (ASF) under one or more
+ * contributor license agreements.  See the NOTICE file distributed with
+ * this work for additional information regarding copyright ownership.
+ * The ASF licenses this file to You under the Apache License, Version 2.0
+ * (the "License"); you may not use this file except in compliance with
+ * the License.  You may obtain a copy of the License at
+ *
+ *    http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+package org.apache.seatunnel.common.utils;
+
+import java.lang.reflect.Method;
+
+public class ReflectionUtils {
+
+    public static Method getDeclaredMethod(Class<?> clazz, String methodName, Class<?>... parameterTypes) {
+
+        Method method;
+
+        for (; clazz != Object.class; clazz = clazz.getSuperclass()) {
+            try {
+                method = clazz.getDeclaredMethod(methodName, parameterTypes);
+                method.setAccessible(true);
+                return method;
+            } catch (Exception e) {
+                // do nothing
```

Comment:
I think we may need to log the exception here, if you think this exception is not important, we can just log in `warn` level.

Reply:
This logic is in the loop. Throw Execption is very normal thing. Because throw Execption so it can go on search method.

Reply:
Ok, I see the code, when the method is not exist, there will throw the `NoSuchMethodException`, you need to catch it and find from parent, so for other exception, you don't eat it.
Suggest to change the implementation to below:
```java
@Nullable
    public static Method findMethod(Class<?> clazz, String name, @Nullable Class<?>... paramTypes) {
        Assert.notNull(clazz, "Class must not be null");
        Assert.notNull(name, "Method name must not be null");

        for(Class searchType = clazz; searchType != null; searchType = searchType.getSuperclass()) {
            Method[] methods = searchType.isInterface() ? searchType.getMethods() : getDeclaredMethods(searchType, false);
            Method[] var5 = methods;
            int var6 = methods.length;

            for(int var7 = 0; var7 < var6; ++var7) {
                Method method = var5[var7];
                if (name.equals(method.getName()) && (paramTypes == null || hasSameParams(method, paramTypes))) {
                    return method;
                }
            }
        }

        return null;
    }

    private static boolean hasSameParams(Method method, Class<?>[] paramTypes) {
        return paramTypes.length == method.getParameterCount() && Arrays.equals(paramTypes, method.getParameterTypes());
    }
```
The code is copy from spring :).

Reply:
Seem like Spring's reflect method is complex than ours. But if we method is enough, I think maybe change it when when need change it.

Reply:
I am not sure if catch `Exception` here is OK, since the `getMethod` will throw two checkedException `NoSuchMethodException` and `SecurityException`, ignore all exception is not a good way.